aten/src/ATen/native/cuda/BatchLinearAlgebra.cu

```diff
@@ -1,5 +1,6 @@
 #include <ATen/Context.h>
 #include <ATen/cuda/CUDAContext.h>
+#include <ATen/cuda/CUDASolver.h>
```

Comment:
This include is not needed?

Reply:
It was defining the 64 bit macros but I'll remove it now.